codegen/src/main/scala/Generator.scala

```diff
@@ -0,0 +1,70 @@
+package codegen
+
+import java.net.{ URL, URLClassLoader }
```

Comment:
A bunch of imports are no longer used, can you clean them up?

Reply:
What about using `-Xfatal-warnings` and `-Ywarn-unused`?

Reply:
Good point!